docs/tutorials/using_apis/connectivity/mesh_tutorial.md

```diff
@@ -98,9 +88,10 @@ Connect to the mesh network:
     }
 ```
 
-The application configuration file `mbed_app.json` defines the mesh network type (Wi-SUN, Thread or 6LoWPAN-ND), RF radio shield and other configuration parameters.
+The application configuration file `mbed_app.json` defines the mesh network type (Wi-SUN or 6LoWPAN-ND), RF radio shield and other configuration parameters.
 
 Below is an example configuration file for a Thread device connected to an Atmel RF AT86RF233 shield:
```

Comment:
If this supports non-thread, how do I phrase it? Or is it a completely different topic?

Reply:
Replace Thread with Mesh device we can run any mesh on top any radio so it is ok

Reply:
See commit: https://github.com/ARMmbed/mbed-os-5-docs/pull/1310/commits/90a0d57aa020807d6aff4a7a16be288f0787111e